src/main/kotlin/com/exactpro/th2/rptdataprovider/entities/configuration/Configuration.kt

```diff
@@ -48,6 +48,7 @@ class CustomConfigurationClass {
     val messageUnpackerOutputMessageBuffer: Int = 100
     val messageFilterOutputMessageBuffer: Int = 100
     val messageMergerOutputMessageBuffer: Int = 10
+    val messageIdsLookupLimit: Int = 60 * 30 * 1_000
```

Comment:
question: What is the unit here? Seconds, milliseconds, days? How user is supposed to understand what to use here?

Reply:
note: I don't believe anybody would be interested in using milliseconds for the limit... Why don't use days instead? It seem much more reasonable